### PR TITLE
Add ScalarDB Cluster support

### PR DESCRIPTION
**Pull request by brfrn169**
This PR just adds ScalarDB Cluster support. Please take a look!

### CODE REVIEW

File: scalardb-test/build.gradle [hunk 1]
```diff
@@ -43,6 +50,7 @@ dependencies {
     implementation group: 'com.scalar-labs', name: 'scalardb-sql', version: '4.0.0-SNAPSHOT'
     implementation group: 'com.scalar-labs', name: 'scalardb-sql-jdbc', version: '4.0.0-SNAPSHOT'
     implementation group: 'com.scalar-labs', name: 'scalardb-graphql', version: '4.0.0-SNAPSHOT'
+    implementation group: 'com.scalar-labs', name: 'scalardb-cluster-client', version: '4.0.0-SNAPSHOT'
```
Comment (brfrn169) [inline scalardb-test/build.gradle:53]: Just added the `scalardb-cluster-client` dependency.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**